utils.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "unicode"
+
+func ToSnake(in string) string {
```

Comment:
Courtesy of https://gist.github.com/elwinar/14e1e897fdbe4d3432e1 and already tested